saleor/graphql/account/mutations.py

```diff
@@ -545,12 +547,14 @@ def perform_mutation(cls, root, info, address_id, user_id, type):
             raise ValidationError({
                 'address_id': 'The address doesn\'t belong to that user.'})
 
-        if type == AddressTypeEnum.BILLING.value:
+        raw_address_type = data.get('type')
+        if raw_address_type == AddressTypeEnum.BILLING.value:
             address_type = AddressType.BILLING
-        elif type == AddressTypeEnum.SHIPPING.value:
+        elif raw_address_type == AddressTypeEnum.SHIPPING.value:
             address_type = AddressType.SHIPPING
         else:
-            raise ValueError('Unknown value of AddressTypeEnum: %s' % type)
+            raise ValueError(
```

Comment:
Consider dropping this `else`, as an execution should never reach there because enums are validated by GraphQL itself.